Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -2043,6 +2047,8 @@ void HTMLMediaElement::executeCueEnterOrLeaveAction(TextTrackCue& cue, CueAction
     scheduleEventOn(cue, WTFMove(event));
 }
 
+=======
+>>>>>>> parent of 969ac99ba7af (AX: Consider VTT-based audio descriptions with text-to-speech.)
```

Comment:
The entire `HTMLMediaElement::setSpeechSynthesisState` method should be deleted.